pkg/network/ebpf/c/prebuilt/usm.c

```diff
@@ -49,7 +49,8 @@ int BPF_KPROBE(kprobe__tcp_sendmsg, struct sock *sk) {
 }
 
 SEC("tracepoint/net/netif_receive_skb")
-int tracepoint__net__netif_receive_skb(struct pt_regs* ctx) {
+int tracepoint__net__netif_receive_skb(void *ctx) {
+    CHECK_BPF_PROGRAM_BYPASSED()
```

Comment:
Because we need to leave the arguments untouched (versus casting from `ctx` fields for other program types), I don't think it is possible to have the same macro signature. I would either have to omit the opening curly brace, or wrap the entire function with the macro.